### DISCUSSION TITLE
Update a bootstrapper application using msi package without duplicates in ARP window

### DISCUSSION THREAD

**Discussion by EvgenPervenenko** [wixtoolset/issues#6825] [Questions]
Hi, all, I have faced with issue. When I trying to update burn bootstapper msi package, using .msi newer version(.msi has the same upgrade code as old .msi) I have duplicates in ARP window, are there any workaround

Note: I want install my app using .msi and .exe and upgrade using .msi and .exe

**Comment by rseanhall** [comment]
The purpose of the ARP window is to allow the user to uninstall programs. Bundles will only uninstall MSIs that it has installed. So if you install a bundle and then manually upgrade one the MSIs that was installed by that bundle, then uninstalling the bundle will not uninstall that upgraded MSI. The behavior here is correct because it is allowing the user to uninstall everything that was installed. For the general case, there is no workaround. Always update the contents of a bundle with a bundle.

For simple bundles with only one .msi, then you could theoretically work around this by making the .msi uninstall the bundle from a custom action. There is no built-in support for accomplishing this today.

**Comment by EvgenPervenenko** [comment]
@rseanhall thanks for your reply, a little question about theoretically workaround, is it possible to uninstall a bundle in silent mode(without user efforts)?

**Reply by robmen** [reply]
Yes. You can pass the `-quiet` command-line switch along with the `-uninstall` switch to quietly uninstall a bundle.

**Comment by EvgenPervenenko** [comment]
@rseanhall @robmen thanks guys for your help, will try
